notebooks/concepts/imviz_compass_mpl.ipynb

```diff
@@ -0,0 +1,578 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9a8d6eac-6626-415d-ade4-3f986cf18a69",
+   "metadata": {},
+   "source": [
+    "# Imviz compass visualization using Matplotlib\n",
+    "\n",
+    "This concept notebook visualizes the compass algorithm that Imviz will use. For simplicity, visualization is done using Matplotlib.\n",
+    "\n",
+    "Some of these examples are also in the unit tests for `wcs_utils.py`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e571e723",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import gwcs\n",
+    "import numpy as np\n",
+    "from astropy import units as u\n",
+    "from astropy.coordinates import ICRS\n",
+    "from astropy.io import fits\n",
+    "from astropy.modeling import models\n",
+    "from astropy.utils.data import download_file\n",
+    "from astropy.wcs import WCS\n",
+    "from glue.core import Component, Data\n",
+    "from gwcs import coordinate_frames as cf\n",
+    "\n",
+    "from jdaviz import Imviz\n",
+    "from jdaviz.configs.imviz import wcs_utils"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fd58f8ef",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3779ae77-97fa-49db-85e5-7e8213c0cd30",
+   "metadata": {},
+   "source": [
+    "## Case 1: Dummy WCS with 10x10 image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ae33669d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a = np.arange(100).reshape((10, 10))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c334e59e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w = WCS()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "20605e33",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "_ = wcs_utils.draw_compass_mpl(a, w, show=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e63db00-18a4-455c-b683-94659dc25895",
+   "metadata": {},
+   "source": [
+    "Now, let's try that in the Compass plugin. We will display this same image in Imviz. Open the Compass plugin to see the result."
```

Comment:
| This PR | Ginga |
| --- | --- |
| ![dummy_wcs_imviz](https://user-images.githubusercontent.com/2090236/145664980-2d09436b-d6d9-455d-a41d-747f13ffb104.jpg) | ![dummy_wcs_ginga](https://user-images.githubusercontent.com/2090236/144470417-c06d92e1-9d73-4957-b422-b03ff4e4166a.jpg) |